apisix/plugins/opa.lua

```diff
@@ -94,8 +95,27 @@ function _M.access(conf, ctx)
         return 503
     end
 
-    if not data.result then
-        return 403
+    local result = data.result
+
+    if not result.allow then
+        if result.headers then
+            core.response.set_header(result.headers)
+        end
+
+        local status_code = 403
+        local reason = ""
```

Comment:
Is it okay to use `local reason` to define the default value as nil? In addition, would it be better to put the definition of `reason` above `if result.reason then`?

Reply:
Got it, I agree with it. changed